Dockerfile

```
@@ -6,7 +6,7 @@ COPY . /ec
 ARG TARGETOS TARGETARCH
 RUN --mount=type=cache,target=/root/.cache/go-build \
     --mount=type=cache,target=/go/pkg \
-    GOOS=${TARGETOS} GOARCH=${TARGETARCH} GO111MODULE=on CGO_ENABLED=0 go build -ldflags "-X main.version=$(cat VERSION | tr -d '\n')" -o bin/ec ./cmd/editorconfig-checker/main.go
+    GOOS=${TARGETOS} GOARCH=${TARGETARCH} GO111MODULE=on CGO_ENABLED=0 go build -ldflags "-X main.version=$(cat VERSION | tr -d '\n')" -o bin/editorconfig-checker ./cmd/editorconfig-checker/main.go
```

`GO111MODULE=on` is no longer relevant since Go 1.17:

```suggestion
    GOOS=${TARGETOS} GOARCH=${TARGETARCH} CGO_ENABLED=0 go build -ldflags "-X main.version=$(cat VERSION | tr -d '\n')" -o bin/editorconfig-checker ./cmd/editorconfig-checker/main.go
```

---

I would not do this within the scope of this PR. We could remove relict that well before this breaking change gets merged.

---

Yeah no worries, I just noticed it when glancing over the diff and thought I'd bring it to your attention 😅 (_Go 1.17 was released in Aug 2021, so it's been irrelevant for a while_)